### DISCUSSION TITLE
AllThePlaces and StreetComplete

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#5957] [General]
I have a project to try using [AllThePlaces](https://github.com/alltheplaces/alltheplaces) in StreetComplete - basically, a new quest asking "have you seen this POI here"?

My proposal for quest icon would be a radar symbol, and have not done much more as far as SC implementation goes.

But I spend significant time on processing OpenStreetMap and AllThePlaces data to list POI candidates. And in fact I have now site showing what appears missing in OSM but is listed as existing by ATP. See https://matkoniecz.codeberg.page/improving_openstreetmap_using_alltheplaces_dataset/

If anyone is interested on what I spend time recently (and was less active in core StreetComplete project) you are welcome! And please, try to find some bad data and complain about it! If it would be used in StreetComplete it would be nice to spot and hide/fix data that is systematically broken.

More info and a discussion thread at https://community.openstreetmap.org/t/improving-openstreetmap-shop-coverage-with-alltheplaces/119979

**Comment by mnalis** [comment]
- Also related: https://github.com/Helium314/SCEE/issues/522

**Comment by pktiuk** [comment]
It could be also used for filling missing data for buildings. For example it could autofill opening hours for shops based on ATP

**Comment by matkoniecz** [comment]
Yes,  I have something mostly done doing that.

**Reply by pktiuk** [reply]
Do you mean this PR: https://github.com/streetcomplete/StreetComplete/pull/6302 ?    
Do you have any ETA for for finishing it? (This year?)

**Reply by matkoniecz** [reply]
yes, among others

it depends a bit on other projects and I hope for earlier than this year, but this year is the worst planned case

I hope to get it finished before PR turns one year old
